homeassistant/components/auth/indieauth.py

```diff
@@ -81,8 +80,22 @@ def handle_starttag(self, tag, attrs):
                     if chunks == 10:
                         break
 
-    except (asyncio.TimeoutError, ClientError) as ex:
-        _LOGGER.error("Error while looking up redirect_uri %s: %s", url, ex)
```

Comment:
Is the error message `ex` not descriptive enough?

Reply:
There's no string version of `ClientError` sadly, so nothing is output for that bit.